src/content/en/ilt/pwa/e-commerce-lab-3-payment-request-api.md

```diff
@@ -442,7 +442,7 @@ request.addEventListener('shippingoptionchange', e => {
 
 #### Explanation
 
-Upon receiving the `shippingoptionchange` event, the `request` object's `shippingOption` is updated. The `shippingOption`It indicates the `id` of the selected shipping options. The `id` is passed to `buildPaymentsDetails`, which looksLook for the price of the shipping option and updates the display items so that the user knows the total cost is changed. `buildPaymentsDetails alsoAlso` changes the shipping option's `selected` property to `true` to indicate that the user has chosen the item. Checkout `buildPaymentDetails()` function in __app/scripts/modules/payment-api.js__ to see how it works.
```

Comment:
This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc